### PR TITLE
New chart: Apache ActiveMQ

### PR DESCRIPTION
**Pull request by algchoo**
## Changes
* [created activemq deployment with loadgen](https://github.com/observIQ/charts/commit/3958c05c2e8acd06489f33af0529e845ac2e066f)
* [created dockerfile to build activemq loadgen container](https://github.com/observIQ/charts/commit/c16ee7d887e0ada2532988b14bb72e4f8d8b663c)
* [updated deployment to pull loadgen image from values yaml](https://github.com/observIQ/charts/commit/23295967fbf0e81f7e424dd7eac7010b76f7e655)

## Description of Changes

> There will be a follow up PR to update the image being pulled by the cronJob for load generation, as the image needs to be built by CI before we can include it here.

These changes aim to include a new chart for Apache ActiveMQ with load generation to be used as a sample application in the `grafana/cloud-onboarding` repository.

The deployment is a single instance of ActiveMQ behind a service. The load generation is a cronjob that utilizes the container being added by this PR. The container is built using a multi-stage Dockerfile that builds a client to produce messages for a queue and topic every so often. The module being used in the Go files is [Azure/amqp-go](https://pkg.go.dev/github.com/Azure/go-amqp#ReceiveOptions).

## **Please check that the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] CI passes

### CODE REVIEW

File: charts/apache-activemq/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+apiVersion: v2
+appVersion: 5.18.2
+engine: gotpl
+home: https://github.com/observIQ/charts/tree/main/charts/aerospike
```
Comment (jsirianni) [inline charts/apache-activemq/Chart.yaml:4]: aerospike

File: charts/apache-activemq/values.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+image: apache/activemq-classic:5.18.2
+loadGeneratorImage: bmedora/activemq-loadgen:test123
```
Comment (jsirianni) [inline charts/apache-activemq/values.yaml:2]: We should use the image built by this repo's ci/cd

File: container/apache-activemq-loadgen/receiver-producer.go [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+		// Create a sender
+		sender, err := session.NewSender(ctx, queue, nil)
+		failOnError(err, "Failed to create new sender")
+		ctx, cancel := context.WithTimeout(ctx, 5*time.Second)
+
+		for i := 0; i < count; i++ {
+			// Send message
+			err = sender.Send(ctx, amqp.NewMessage([]byte("Hello World!")), &amqp.SendOptions{})
+			failOnError(err, "Failed to send message")
+		}
```
Comment (jsirianni) [inline container/apache-activemq-loadgen/receiver-producer.go:54]: Do you want this to happen super fast or should it sleep between requests?

File: container/apache-activemq-loadgen/receiver-producer.go [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+		// Create a sender
+		sender, err := session.NewSender(ctx, topic, nil)
+		failOnError(err, "Failed to create new sender")
+		ctx, cancel := context.WithTimeout(ctx, 5*time.Second)
+
+		for i := 0; i < count; i++ {
+			// Send message
+			err = sender.Send(ctx, amqp.NewMessage([]byte("Hello World!")), &amqp.SendOptions{})
+			failOnError(err, "Failed to send message")
+		}
```
Comment (jsirianni) [inline container/apache-activemq-loadgen/receiver-producer.go:88]: Same as previous comment

File: charts/apache-activemq/Chart.yaml (hunk 1 above)
Comment (algchoo) [reply charts/apache-activemq/Chart.yaml]: Thanks for catching this.

File: charts/apache-activemq/values.yaml (hunk 2 above)
Comment (algchoo) [reply charts/apache-activemq/values.yaml:2]: There will be a follow up PR, as noted in the description.

**Review by jsirianni — APPROVED** [review]

**Pull request merged**